tests/unittest/_torch/visual_gen/test_offloading.py

```
@@ -0,0 +1,443 @@
+
+    pipeline.cleanup()
+
+
+def test_initialize_rejects_configured_stage_parts_that_are_unavailable_for_model():
+    pipeline = _CustomOffloadPipeline(
+        _make_config(offload_stages=[["denoising_transformer", "denoising_transformer_2"]])
+    )
+
+    with pytest.raises(
+        ValueError,
+        match="Unknown offload_config.stages entries.*denoising_transformer_2.*denoising_transformer",
```

Ruff (0.15.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)